local/kcp/start.sh

```diff
@@ -40,7 +40,8 @@ kcp-binaries() {
     git clone https://github.com/kcp-dev/kcp.git
     KCP_DIR="${KCP_PARENT_DIR}/kcp"
     pushd kcp
-    KCP_BRANCH="${KCP_BRANCH:-release-0.5}"
+    kcp_default_version=$(yq '.images[] | select(.name == "kcp") | .newTag' "$PARENT_PATH/../../ckcp/openshift/overlays/dev/kustomization.yaml")
+    KCP_BRANCH="${KCP_BRANCH:-$kcp_default_version}"
     git checkout "${KCP_BRANCH}"
     make build
     popd
```

Comment:
under line 92: kubectl --kubeconfig="${KUBECONFIG}" config use-context root
it is now needed to navigate back to the root workspace:
~~~
KUBECONFIG="${KUBECONFIG}" kubectl kcp workspace use root
~~~

Reply:
Done.